### PR TITLE
Add support for infinite timeoute polling and update plugin.yaml to i…

### PR DESCRIPTION
**Pull request by mabuaisha**
…nclude a default value for timeout

### CODE REVIEW

**Comment by EarthmanT** [comment]
#73 @isaac-s

File: plugin.yaml [hunk 1]
```diff
@@ -252,6 +252,8 @@ node_types:
           inputs:
             workflow_id:
               default: install
+            timeout:
+              default: 120
```
Comment (isaac-s) [inline plugin.yaml]: The current default timeout is `900` (see `constants.EXECUTION_TIMEOUT`). Why are we making it `120`?

Comment (mabuaisha) [reply plugin.yaml]: @isaac-s Yeah I know it is 900 but I just add it here to make sure  this update to ``plugin.yaml make`` sense and does not confusing the end user and if we really need to add anything else.
please let me know what do you think

**Comment by mabuaisha** [comment]
@isaac-s Yeah sure I did notice that and I do not know why it is written on this way because it will be always evaluated as 900.

I did update the retrieval of ``timeout``

**Pull request merged**